src/settings/tabrepository.cpp

```diff
@@ -0,0 +1,12 @@
+#include "tabrepository.h"
+#include <QtWidgets>
+
+TabRepository::TabRepository(QWidget *parent) : QWidget(parent)
```

Comment:
Header QtWidgets includes all elements of Qt UI
It is huge
You must remove this include from your files, especially - from headers
**_It might cause identifiers conflict_**

Reply:
removed